articles/openshift/oshift-how-backup-migrate.md

```diff
@@ -20,17 +20,17 @@ toc_mdlink: oshift-how-backup-migrate.md
 
 ### Overview
 
-Velero is an open-source tool set that provides the ability to backup and restore kubernetes resources. The GitHub repository is avaialble at: <https://github.com/vmware-tanzu/velero>.
+Velero is an open-source tool set that provides the ability to backup and restore Kubernetes resources. The GitHub repository is available at: <https://github.com/vmware-tanzu/velero>.
 
 This guide outlines how to:
 
 - Install Velero in your cluster (using our Elastic Cloud Storage product as the backup location)
 
-- Back up and restore namespaces to the same cluster
+- Backup and restore namespaces to the same cluster
 
 - Restore a namespace to a new cluster
 
-Velero offers a tool called restic that enables backup of persistent volume data regardless of provider, however this guide doesn't cover restic as the tool is considered beta-quality. If you'd like to test it you can find more on restic here: <https://velero.io/docs/v1.1.0/restic/>.
```

Comment:
Looks like restic actually doesn't use an uppercase r (it's important with product names that we use the exact spelling used by whoever makes it)